### PR TITLE
[14.0] [MIG] account_invoice_mass_sending

### PR DESCRIPTION
**Pull request by jguenat**
Standard migration

I want to propose an improvement :
Instead of a new action "Mass Send & Print" I want to inherit the standard Send&Print wizard and add a new button "Enqueue Send". This would allow to select a mail template and would be more intuitive to use (inspired by account_invoice_validation_queued)
What do you think @RobinetDenisAcsone @ThomasBinsfeld ?

If yes what would be the procedure, add a commit for the improvement in this PR or open a new one?

Thanks

### CODE REVIEW

**Comment by ThomasBinsfeld** [comment]
Hello @jguenat, i think you should first migrate the module and then open an improvement pull request.

**Comment by jguenat** [comment]
Hi @BurkhalterY ,
Can you review this one?
thx

File: account_invoice_mass_sending/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    def do_prepare_send_print(self):
+        for rec in self:
+            rec.with_delay().do_send_print()
+
+    def do_send_print(self):
+        for rec in self:
+            if not rec.partner_id.email:
+                raise UserError(
+                    _("Missing email address on customer " "'{customer_name}'.").format(
```
Comment (BurkhalterY) [inline account_invoice_mass_sending/models/account_move.py]: The syntax can be simplified, maybe there was two lines before.

**Review by BurkhalterY — APPROVED** [review]
Except that, it looks good to me.
Thank you !

**Review by petrus-v — APPROVED** [review]
code review

LGTM

File: account_invoice_mass_sending/models/account_move.py (hunk 1 above)
Comment (jguenat) [reply account_invoice_mass_sending/models/account_move.py]: done :)

**Comment by jguenat** [comment]
Dear @OCA/accounting-maintainers,
Is there something I can do to get this merged?
Thank you

**Review by ThomasBinsfeld — APPROVED** [review]

**Review by TDu — APPROVED** [review]

**Comment by rousseldenis** [comment]
@OCA/accounting-maintainers @OCA/core-maintainers

**Pull request merged**